.gitignore

```diff
@@ -1,10 +1,10 @@
-**/*.pyc
+**/*.egg-info/*
+**/.idea/*
+**/.nox/*
+**/.pytest_cache/*
 **/__pycache__/*
-*.egg-info/*
-.cache/*
-.coverage
-.idea/*
-.nox/*
-build/*
-dist/*
-venv/*
+**/dist/*
+**/venv*/*
```

Comment:
This is copied from gcloud-aio. I just added a start here because I'm using two venvs.